templates/shortcodes/clients_showcase.html

```diff
@@ -7,9 +7,7 @@
         <img src="thumbnails/{{ client.thumbnail }}" alt="{{ name }} Preview">
         <h3>{{ name }}</h3>
         <p>{{ client.summary | safe }}</p>
-        <div class="platforms-links">
-            {{ clients::platform_links(packages=client.packages) }}
-        </div>
+        {{ clients::platform_links(packages=client.packages) }}
```

Comment:
You may need a div around this, to be able to turn it into a row. At the moment it displays all of the platforms vertically, likely because of the `flex-direction: column;` of its direct `.featured-client-card` parent.

Reply:
I didn't understand what could cause the logos to go out of the cards, but it might very well be the platform links being vertically displayed instead of horizontally displayed

![image](https://user-images.githubusercontent.com/1688309/214803789-64dd4a33-36f4-4290-80da-a38916402754.png)

Reply:
Huh. That's not what this looks like in my browser 😱 in mine it is like in the mockup. That looks like I am using something that your browser doesn't support?

Reply:
I'm using `Firefox 109.0 (64-bit)` on `macOS Ventura 3.1 (22C65)`, nothing too fancy or exotic, and no WebExtension fiddling with the content (uBlock Origin would be the most likely to, but blocked 0% of requests on this page)

Reply:
That's weird 🤔 on my Firefox it looked fine when writing it. 😱 That's weird.